docusaurus.config.ts

```diff
@@ -0,0 +1,133 @@
+import { themes as prismThemes } from "prism-react-renderer";
+import type { Config } from "@docusaurus/types";
+import type * as Preset from "@docusaurus/preset-classic";
+import { tailwindPlugin } from "./plugins/tailwind";
+
+const config: Config = {
+    title: "Testplane Docs",
+    tagline: "Testplane is a modern end-to-end testing framework.",
+    favicon: "img/favicon.ico",
+
+    // Set the production url of your site here
+    url: "https://testplane.example.com",
+    // Set the /<baseUrl>/ pathname under which your site is served
+    // For GitHub pages deployment, it is often '/<projectName>/'
+    baseUrl: "/",
+
+    // GitHub pages deployment config.
+    organizationName: "gemini-testing",
+    projectName: "testplane-docs",
+
+    onBrokenLinks: "throw",
+    onBrokenMarkdownLinks: "warn",
+
+    i18n: {
+        defaultLocale: "en",
+        locales: ["en"],
+    },
+
+    presets: [
+        [
+            "classic",
+            {
+                docs: {
+                    sidebarPath: "./sidebars.ts",
+                    editUrl: "https://github.com/gemini-testing/testplane-docs/tree/main/docs/",
+                },
+                blog: {
+                    showReadingTime: true,
+                    editUrl: "https://github.com/gemini-testing/testplane-docs/tree/main/blog/",
+                },
+                theme: {
+                    customCss: "./src/scss/custom.scss",
+                },
+            } satisfies Preset.Options,
+        ],
+    ],
+
+    themeConfig: {
+        image: "img/docusaurus-social-card.jpg",
+        navbar: {
+            title: "testplane",
+            logo: {
+                alt: "Testplane Logo",
+                src: "img/logo.svg",
```

Comment:
The logo is a hybrid of a dinosaur and floppy disk. Is it just "for now", or it is the final logo?

Reply:
It's a default one, provided by docusaurus, till we have our own.